pkg/print/fs-lvp-velero-config.go

```diff
@@ -92,7 +92,7 @@ func printLvpFileSystemVeleroInstructions(c *LvpFileSystemVeleroConfig, log *log
 	  replicated.com/nfs: ObjectStore
 	  replicated.com/hostpath: ObjectStore
 	data:
-	  fileserverImage: %s/local-volume-fileserver:v0.1.0
```

Comment:
so this is part of the `local-volume-provider` image now?

Reply:
it is, yes

Reply:
Or at least the v0.2.0+ versions of `local-volume-provider` no longer include that image, and the velero plugin in kurl doesn't use it